rs/execution_environment/src/execution_environment/tests.rs

```diff
@@ -3170,30 +3170,59 @@ fn test_sign_with_schnorr_api_is_enabled() {
 }
 
 #[test]
-fn test_vetkd_public_key_api_is_disabled() {
+fn test_vetkd_public_key_api_is_enabled() {
```

Comment:
Would it make sense to add a test that all cycles are refunded?

Reply:
You mean to add some Cycles to the call and then seeing whether `response.refund` contains that value?

Reply:
I'm not so sure, I don't see this check for any other `IC00Method`, so I didn't implement it.